### PR TITLE
adding placeholder option to optionally add KVM packages

### PR DESCRIPTION
**Pull request by heckj**
coordinates with https://github.com/RackHD/on-http/pull/67 to enable adding KVM to CentOS/RHEL installs

### CODE REVIEW

**Comment by benbp** [comment]
:+1:

File: spec/lib/jobs/message-cache-job-spec.js [hunk 1]
```diff
@@ -108,7 +108,8 @@ describe("Message Cache Job", function () {
             this.sandbox.stub(job, 'createSetIpmiCommandResultCallback').returns('fn return stub');
             job._run();
             expect(job._subscribeIpmiCommandResult.callCount).to.equal(5);
-            _.forEach(['sdr', 'selInformation', 'sel', 'chassis', 'driveHealth'], function(command) {
+            _.forEach(['sdr', 'selInformation', 'sel', 'chassis', 'driveHealth'],
```
Comment (heckj) [inline spec/lib/jobs/message-cache-job-spec.js]: safely ignorable - loaded in spec helper as a global lodash instance for tests

**Comment by yyscamper** [comment]
:+1:

File: spec/lib/jobs/ansible-job-spec.js [hunk 2]
```diff
@@ -3,8 +3,7 @@
 
 'use strict';
 
-var uuid = require('node-uuid'),
-    events = require('events');
+var uuid = require('node-uuid');
```
Comment (yyscamper) [inline spec/lib/jobs/ansible-job-spec.js]: Seems this file change doesn't belong to this commit as not relation with kvm. How about move it to another commit?

Comment (heckj) [inline spec/lib/jobs/ansible-job-spec.js]: I think you just wanted to see if I could rebase a single commit back out into multiple commit. Never one to stand down from a dare... Viola!

**Comment by heckj** [comment]
For future reference, the 4GB Installation ISO that I used to provide my testing repository doesn't include kvm or python-virtinst, but otherwise I've verified these combined patches work to provide an optional KVM installation, ready to merge!
